File: Study-Code/Upcoder/11_Grade/TienDien.cpp
#include <iostream>
using namespace std;

int main()
{
    int n;
    cin >> n;
    
    if(n >= 200)
        cout << "CAT DIEN";
    else if(n < 0)
        cout << "NHAP SAI";
    else if(n <= 50)
        cout << n*2500;
    else if(n > 50 && n <= 100)
        cout << 50 * 2500 + (n - 50) * 3000;
    else if(n > 100 && n <= 200)
        cout << 50 * 2500 + 50 * 3000 + (n - 100) * 5000;
        
    return 0;
}

File: Study-Code/K11_Py/Re_Nhanh/TienDien.py
def calculate_cost(n):
    if n >= 200:
        return "CAT DIEN"
    elif n < 0:
        return "NHAP SAI"
    elif n <= 50:
        return n * 2500
    elif n <= 100:
        return 50 * 2500 + (n - 50) * 3000
    elif n <= 200:
        return 50 * 2500 + 50 * 3000 + (n - 100) * 5000
        
n = int(input())

result = calculate_cost(n)
print(result)
        
